### PR TITLE
Feature/tg 68/list employees page

### PR DESCRIPTION
**Pull request by FefeTroanes**
This is my request to merge this branch for the Trackgenix app.
Here I developed the table where are listed all the employees fetched from de database, each one employee from that list has two buttons:
- One to delete the employee: when someone clicks it, it shows an alert that asks confirmation to delete it in wich affirmative case will delete the employee.
- The other button is to edit an employee, I could not test it properly because the functionality is on another branch but we did a test where we copied the link from the button and pasted it in the browser of the developer of the functionality and it worked.

The page also has a button to add an employee that redirects to the funcionality link (that is also not on this branch), we tested the same way that the previous point and it worked.

### CODE REVIEW

File: src/Components/Employees/List/List.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        ? setEmployees(employees.filter((employee) => employee._id !== id))
+        : alert('Error deleting this employee');
+    }
+  };
+
+  const editForm = (id) => {
+    window.location = `/employees/editEmployee/${id}`;
+  };
+
+  // const EditEmployee = async (id) => {
```
Comment (mpueblas) [inline src/Components/Employees/List/List.js:35]: te quedó comentada esa variable

File: src/Components/Employees/Row/Row.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <td>{fullName}</td>
+      <td>{dob}</td>
+      <td>{phone}</td>
+      <td>{gender}</td>
+      <td>{address}</td>
+      <td>{active ? 'Active' : 'Unactive'}</td>
+      <td>{createdAt}</td>
+      <td>{updatedAt}</td>
+      <td>
+        {/* <RowBtn className="fa-solid fa-pen-to-square" id={id} onEdit={onEdit} employee={employee} /> */}
```
Comment (mpueblas) [inline src/Components/Employees/Row/Row.js:34]: te quedó comentado

File: src/Components/Employees/Row/Row.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <a href={`/employees/editEmployee?id=${id}`}>
+          <i className="fa-solid fa-pen-to-square"></i>
+        </a>
+      </td>
+      <td>
+        {/* <RowBtn className="fa-solid fa-xmark" id={id} onDelete={onDelete} employee={employee} /> */}
```
Comment (mpueblas) [inline src/Components/Employees/Row/Row.js:40]: comentado acá

File: src/Components/Employees/RowBtn/RowBtn.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// const RowBtn = ({ className, onDelete, employee }) => {
+//   return (
```
Comment (mpueblas) [inline src/Components/Employees/RowBtn/RowBtn.js:2]: comentado.

File: src/Components/Employees/index.js [hunk 5]
```diff
@@ -1,25 +1,14 @@
-import { useEffect, useState } from 'react';
+// import { useEffect, useState } from 'react';
```
Comment (mpueblas) [inline src/Components/Employees/index.js:1]: comentado

**Review by nicolascostanza — APPROVED** [review]

File: src/Components/Employees/AddBtn/AddBtn.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+const AddBtn = () => {
+  return (
+    <a href="http://localhost:3000/employees/addEmployee">
```
Comment (javierlazzarini) [inline src/Components/Employees/AddBtn/AddBtn.js:3]: Hola Fede, fijate que Fernando me hizo una correccion para que haya consistencia en las rutas y tuve que cambiarla, aca te dejo la modificacion para que la pases en el boton.
case '/employee-add':

**Comment by FlorenciaEusebi** [comment]
##Test Results
Tested locally on alfonso-trackgenix-server.vercel.app

**Test Results: Passed**
Bugs found:

1. The Employees' table should look smaller in order to fit the screen.
bug 1

2. A "Back" button to go to the main menu is suggested.
BUG 2

**Review by FlorenciaEusebi — COMMENTED** [review]
Please refer to https://github.com/BaSP-m2022/alfonso-trackgenix-app/pull/3#issuecomment-1146453452

**Pull request merged**